apps/dashboard/app/models/project_readme_markdown_renderer.rb

```diff
@@ -0,0 +1,69 @@
+# Utility class for rendering markdown README files for projects.
+class ProjectReadmeMarkdownRenderer < Redcarpet::Render::HTML
```

Comment:
Can you tell me why we need this class? I know we render markdown elsewhere, and I don't think it's as complex as this is.

Reply:
Looking elsewhere, there are similar classes that seem to handle some configuration (like extensions and inserting images) but if we render `@content` in the model or somewhere similar then this logic could probably move there as well.